src/definitions/behaviors/visibility.js

```diff
@@ -246,8 +246,8 @@
                     ;
                     while (imagesLength--) {
                         cacheImage = document.createElement('img');
-                        cacheImage.onload = handleLoad;
-                        cacheImage.onerror = handleLoad;
+                        cacheImage.addEventListener('load', handleLoad);
+                        cacheImage.addEventListener('error', handleLoad);
```

Comment:
eslint error:
```
Prefer `addEventListener` over `onerror`. Note that there is difference between
`{window,element}.onerror` and `{window,element}.addEventListener('error')`
```

Reply:
As the attached `handleLoad` function has no parameters and does not need to access event properties, this change is fine.
window.onerror would expect a different parameter list.